<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>811</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(C:/Users/Dell/Downloads/bulldozerBack.jfif);</string>
   </property>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adminButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Bulldozer Price Prediction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>345</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Expected Meter Reading</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MeterReading">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter ModelID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="YearMade">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1990</number>
    </property>
    <property name="maximum">
     <number>2021</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="modelID">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>30000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Year Made</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/Dell/Downloads/file.qrc"/>
 </resources>
 <connections/>
</ui>
